packages/kori-example/src/usage.ts

```diff
@@ -75,6 +75,55 @@ const app = createKori({
   requestValidator: createKoriZodRequestValidator(),
   responseValidator: createKoriZodResponseValidator(),
   loggerFactory,
+
+  // Pre-validation error handler (Content-Type, JSON parsing errors)
+  onPreRequestValidationError: (ctx, err) => {
```

Comment:
It would be better to put error handlers in the route definition rather than the instance definition as examples.

Reply:
@cursoragent It would be better to put error handlers in the route definition rather than the instance definition as examples.